Source/glTFRuntime/Private/glTFRuntimeParserStaticMeshes.cpp

```diff
@@ -228,79 +228,69 @@ UStaticMesh* FglTFRuntimeParser::LoadStaticMesh_Internal(TSharedRef<FglTFRuntime
 			LODIndices.AddUninitialized(NumVertexInstancesPerSection);
 
 			// Geometry generation
-			ParallelFor(NumVertexInstancesPerSection, [&](const int32 VertexInstanceSectionIndex)
-				{
-					uint32 VertexIndex = Primitive.Indices[VertexInstanceSectionIndex];
-					LODIndices[VertexInstanceBaseIndex + VertexInstanceSectionIndex] = VertexInstanceBaseIndex + VertexInstanceSectionIndex;
+			for (int32 VertexInstanceSectionIndex = 0; VertexInstanceSectionIndex < NumVertexInstancesPerSection; VertexInstanceSectionIndex++)
```

Comment:
Can you add a comment block `// PIXELPOOL - [explanation]` and maybe even a reference link to explain why we apply this difference compared to the gltf plugin? That way we can easily come back to this later to see if our custom change can be removed

Reply:
Good idea. Done